products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,210 @@
+customsearchtemplatespath: "" # The path to the user's custom search templates file, if it's provided
+debug: false # The CLI's debug setting
+environments: # The CLI's configured environments
+  example:
+    baseurl: https://example.api.identitynow.com
+    pat:
+      accesstoken: example-access-token
+      clientid: example-client-id
+      clientsecret: example-client-secret
+      expiry: example-access-token-expiry
+    tenanturl: https://example.identitynow.com
+```
+You can copy the example into your `config.yaml` file. You must specify your baseurl, tokenurl, clientsecret, and clientid and any other necessary information for your chosen authentication method. 
+
+You can have both authentication methods configured at once, but only one can be active at a time. 
+
+#### Environment variable configuration 
+
+You can also store your configuration in environment variables. This can be useful when you are using the CLI in an automated environment like a continuous integration and continuous deployment (CI/CD) pipeline. In these types of scenarios, consuming the cconfiguration from environment variables would be easier than creating the configuration file. 
+
+To export the environment variables on **Linux/Mac**, open your terminal app and run these commands:
+
+```shell
+export SAIL_BASE_URL=https://{tenant}.api.identitynow.com
+export SAIL_CLIENT_ID={clientID}
+export SAIL_CLIENT_SECRET={clientSecret}
+```
+
+To get your environment variables to persist across terminal sessions, add these exports to your shell profile, something like `~/.bash_profile`.
+
+To store your configuration in environment variables on **Windows**, run Powershell as an administrator and run these commands: 
+
+```powershell
+$env:SAIL_BASE_URL=https://{tenant}.api.identitynow.com
+$env:SAIL_CLIENT_ID={clientID}
+$env:SAIL_CLIENT_SECRET={clientSecret}
+```
+
+To get your environment variables to persist across PowerShell sessions, run this command instead: 
+
+```powershell
+[System.Environment]::SetEnvironmentVariable('SAIL_BASE_URL','https://{tenant}.api.identitynow.com')
+[System.Environment]::SetEnvironmentVariable('SAIL_CLIENT_ID','{clientID}')
+[System.Environment]::SetEnvironmentVariable('SAIL_CLIENT_SECRET','clientSecret}')
+```
+
+## Usage
+
+Run the `sail` command for an overview of the available commands and flags. You can use the `-h` flag with any available command to see additional available options for each command: 
+
+The following commands are available.: 
+- `connectors`: This command is a CLI interface for the SaaS Connectivity platform. The CLI is the best way to create and manage SaaS connectors within your tenant. For more information about the `connectors` command, refer to the CLI [Connectors guide](https://developer.sailpoint.com/idn/tools/cli/connectors). For more information about the SaaS Connectivity platform, refer to [SaaS Connectivity](https://developer.sailpoint.com/idn/docs/saas-connectivity). 
```

Comment:
Is the command connectors or conn?

Reply:
I think I want connectors here since it pulls up the conn commands